skycoin/skywire
# Add the fwd functionality to the API

Did you run `make format && make check`?
Can’t from my dev enviroment, will fix any problem form another machine if a problem appears.

 Changes:	
- Several new endpoints were addded to the Hypervisor API, to be able to access the functionality of the `skywire-cli fwd` and `skywire-cli rev` commands. This will allow to access the functionalities from the UI.

How to test this PR:
Call the API endpoints, they should work as expected.

NOTE: during testing using Postman, when the `POST /visors/{pk}/rev` endpoint was called to create a connection, Postman it always reported that the server did not respond, but the connection was in fact created, which could be checked by calling `GET /visors/{pk}/rev`. This could mean that the operation for creating the connection to the remote port is slow and that there is a timeout in the API that causes problems with that.
